docs/DeveloperGuide.md

```diff
@@ -453,40 +454,44 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
-
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   1. Use the command `java -jar InSUREance.java` to launch the app
+   2. Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
 1. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+   1. Re-launch the app by using the command `java -jar InSUREance.java`<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
 
 ### Deleting a client
 
 1. Deleting a client while all clients are being shown
 
    1. Prerequisites: List all clients using the `list` command. Multiple clients in the list.
 
-   1. Test case: `delete 1`<br>
+   2. Test case: `delete 1`<br>
       Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
 
-   1. Test case: `delete 0`<br>
+   3. Test case: `delete 0`<br>
       Expected: No client is deleted. Error details shown in the status message. Status bar remains the same.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
-
 ### Saving data
 
-1. Dealing with missing/corrupted data files
+1. Dealing with missing data files
+
+   1. Delete `config.json` and re-launch the app. Expected: New `config.json` created. Existing data is not affected.
+
+   2. Delete `preferences.json` and re-launch the app. Expected: New `preferences.json` created. Existing data is not affected. 
+   
+   3. Edit the line `"addressBookFilePath"` : `"data/addressbook.json"` to `"addressBookFilePath"` : `"data/data.json"` and re-launch the app. Expected: App starts on clean slate (i.e. with sample data only). 
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+   4. Delete `data/` or `data/addressbook.json`. Expected: New `data/addressbook.json` created. App starts on clean slate (i.e. with sample data only).
 
-1. _{ more test cases …​ }_
+2. Dealing with corrupted data files 
+   1. Add a new field `"newField" : "newField"` to a client. Expected: All data is lost. The app starts on a clean slate.
+   2. Remove `"remark"` field from a client. Expected: The client with the missing `"remark"` field is lost. The rest of the contacts still exist in the contact list.
```

Comment:
Is this meant to be here 😅 was this outdated from when we did the `remark` command

Reply:
er oops cos i ported it over

Reply:
no worries! thanks for fixing 👍